fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsAccountReadPlatformServiceImpl.java

```diff
@@ -703,7 +703,7 @@ public List<Long> retrieveActiveSavingsAccrualAccounts(Long maxSavingsAccountId,
     @Override
     public List<Long> retrieveActiveSavingAccountsForInterestPosting(Long maxSavingsIdInList, int pageSize) {
         String sql = "select id from m_savings_account where status_enum = 300 and (nominal_annual_interest_rate != 0 or (allow_overdraft = true or account_balance_derived <= 0)) "
-                + " and deposit_type_enum != 200 and id > ? order by id asc limit ?";
+                + " and deposit_type_enum 100 and id > ? order by id asc limit ?";
```

Comment:
what is the essence of the limit when the offset won't be taken into consideration?

Reply:
`id > ?` works as an offset in this case